content/en/docs/reference/command-line-tools-reference/feature-gates/port-forward-websockets.md

```diff
@@ -6,9 +6,9 @@ _build:
   render: false
 
 stages:
-  - stage: alpha
-    defaultValue: false
-    fromVersion: "1.30"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.31"
```

Comment:
The proposed change doesn’t seem accurate. As the feature transitions into the beta phase, we need to add a new stage entry within the stages block labeled ‘beta’, rather than modifying the existing alpha stage to beta. Please see the suggestion below and refer to the documentation [here](https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates) for further information.

```suggestion
  - stage: alpha 
    defaultValue: false
    fromVersion: "1.30"
    toVersion: "1.30"
  - stage: beta 
    defaultValue: true
    fromVersion: "1.31"
```

Reply:
I believe I have addressed the issue you mention. Please let me know what you think.